src/tcp_proxy.c

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2018 Xaptum, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <errno.h>
+#include <netdb.h>
+#include <string.h>
+#include <unistd.h>
+
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <netinet/tcp.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+
+#include "log.h"
+#include "tcp.h"
+
+#define AF_XPT 44
```

Comment:
This is already defined in the `sys/socket.h` header [file](https://github.com/xaptum/linux/blob/73228c69e33d7f6593d1397b6040955865d8c158/include/linux/socket.h#L260), so no need to define it here.

Change all occurrences of `AF_XPT` below to `AF_PSOCK` or `PF_PSOCK`.

@dberliner The kernel header file that defines the